adapters/postgres/postgres_test.go

```diff
@@ -1507,3 +1508,208 @@ func Test_containsAsterisk(t *testing.T) {
+				expect:    `DELETE FROM "database"."schema"."table"`,
+			},
+		},
+		{
+			name: "test_UpdateSQL",
+			args: args{
+				gotByFunc: pg.UpdateSQL("database", "schema", "table", "syntax"),
+				expect:    `UPDATE "database"."schema"."table" SET syntax`,
+			},
+		},
+		{
+			name: "test_DatabaseWhere_Empty",
+			args: args{
+				gotByFunc: pg.DatabaseWhere(""),
+				expect:    statements.DatabasesWhere,
+			},
+		},
+		{
+			name: "test_DatabaseWhere_withRequest",
+			args: args{
+				gotByFunc: pg.DatabaseWhere("testrequestwhere"),
+				expect:    fmt.Sprintf(`%v AND testrequestwhere`, statements.DatabasesWhere),
+			},
+		},
+		{
+			name: "test_DatabaseOrderBy",
+			args: args{
+				gotByFunc: pg.DatabaseOrderBy("order", true),
+				expect:    "order",
+			},
+		},
+		{
+			name: "test_DatabaseOrderBy_Empty_hasCount",
+			args: args{
+				gotByFunc: pg.DatabaseOrderBy("", true),
+				expect:    "",
+			},
+		},
+		{
+			name: "test_DatabaseOrderBy_Empty_nocount",
+			args: args{
+				gotByFunc: pg.DatabaseOrderBy("", false),
+				expect: fmt.Sprintf(`
+ORDER BY
+	%s ASC`, statements.FieldDatabaseName),
+			},
+		},
+		{
+			name: "test_SchemaOrderBy",
+			args: args{
+				gotByFunc: pg.SchemaOrderBy("order", true),
+				expect:    "order",
+			},
+		},
+		{
+			name: "test_SchemaOrderBy_Empty_hasCount",
+			args: args{
+				gotByFunc: pg.SchemaOrderBy("", true),
+				expect:    "",
+			},
+		},
+		{
+			name: "test_SchemaOrderBy_Empty_nocount",
+			args: args{
+				gotByFunc: pg.SchemaOrderBy("", false),
+				expect: fmt.Sprintf(`
+ORDER BY
+	%s ASC`, statements.FieldSchemaName),
+			},
+		},
+		{
+			name: "test_TableWhere",
+			args: args{
+				gotByFunc: pg.TableWhere("requestWhere"),
+				expect:    fmt.Sprintf(`%v AND requestWhere`, statements.TablesWhere),
+			},
+		},
+		{
+			name: "test_TableWhere_Empty",
+			args: args{
+				gotByFunc: pg.TableWhere(""),
+				expect:    statements.TablesWhere,
+			},
+		},
+		{
+			name: "test_TableOrderBy",
+			args: args{
+				gotByFunc: pg.TableOrderBy("order"),
+				expect:    "order",
+			},
+		},
+		{
+			name: "test_TableOrderBy_Empty",
+			args: args{
+				gotByFunc: pg.TableOrderBy(""),
+				expect:    statements.TablesOrderBy,
+			},
+		},
+		{
+			name: "test_SchemaTablesWhere",
+			args: args{
+				gotByFunc: pg.SchemaTablesWhere("requestWhere"),
+				expect:    fmt.Sprintf("%v AND requestWhere", statements.SchemaTablesWhere),
+			},
+		},
+		{
+			name: "test_SchemaTablesWhere_Empty",
+			args: args{
+				gotByFunc: pg.SchemaTablesWhere(""),
+				expect:    statements.SchemaTablesWhere,
+			},
+		},
+		{
+			name: "test_SchemaTablesOrderBy",
+			args: args{
+				gotByFunc: pg.SchemaTablesOrderBy("order"),
+				expect:    "order",
+			},
+		},
+		{
+			name: "test_SchemaTablesOrderBy_Empty",
+			args: args{
+				gotByFunc: pg.SchemaTablesOrderBy(""),
+				expect:    statements.SchemaTablesOrderBy,
+			},
+		},
+	}
+	for _, tc := range testCases {
```

Comment:
I like your implementation, but I miss the test that must break because it is not correct